web/libs/ui/src/lib/simple-card/index.tsx

```diff
@@ -15,13 +15,17 @@ export function SimpleCard({
   } & Omit<HtmlHTMLAttributes<HTMLDivElement>, "title">
 >) {
   const className = cn("bg-transparent", cls);
+  const hasHeaderContent = Boolean(title || description);
+  const contentClass = cn("p-4", { "pt-0": hasHeaderContent });
```

Comment:
wouldn't this add the prefix? i thought p-4 is tailwind css class. should `clsx` be used here instead?